src/app/dashboard-widgets/add-codebase-widget/codebase-item/codebase-item.component.html

```diff
@@ -9,8 +9,7 @@
           <a href="{{codebase.attributes.url}}" target="_blank" class="f8-card-codebase-url truncate">{{codebase.name}}</a>
         </div>
         <div class="list-group-item-text f8-card-list-group-item-text">
-          codebase last updated 03/12/2018
-          <!-- replace with real code -->
+          codebase last updated {{ lastUpdated$ | async | date:'short' }}
```

Comment:
How about just `last updated [...]` as it appears in the codebases widget?

Reply:
Sure, that's probably fine - I doubt the exact string was important for that placeholder.